torch/_inductor/scheduler.py

```diff
@@ -1537,7 +1561,11 @@ def can_fuse(self, node1: BaseSchedulerNode, node2: BaseSchedulerNode):
             not config.aggressive_fusion or node1.is_reduction() or node2.is_reduction()
         ):
             return False  # heuristic not needed for correctness
-
+        # Check for node specific epilogue fusion rules
+        if hasattr(node1, "can_fuse_epilogue") and node1.can_fuse_epilogue(node2):
+            # we don't test it the other way around, since node1<=node2 in topological order
+            # and epilogue fusion is only possible if node1<node2
+            return True
```

Comment:
We have something similar for handing fusion into triton templates.  Can we reuse the same API instead of adding this?

Reply:
A few lines below this check, there is a call to self.can_fuse_vertical that might also be a good point for this, but the code in there is not really applicable. Different rules apply, since the EVT fusion has much more limits at the moment. I will hopefully be able to relax more and more of these restrictions in followup PRs, but I do not see a way of doing this without introducing at least a minimal change like this one to scheduler.py